sign_language_datasets/datasets/bsl_corpus/bsl_corpus_utils.py

```diff
@@ -0,0 +1,493 @@
+import re
+import io
+import contextlib
+import requests
+import os
+import pympi
+
+import tensorflow_datasets as tfds
+
+import tensorflow as tf
+from tensorflow_datasets.core.download import downloader
+from tensorflow_datasets.core import utils
+from tensorflow_datasets.core import units
+from tensorflow_datasets.core.download import checksums as checksums_lib
+
+from requests.adapters import HTTPAdapter, Retry
+from typing import Optional, List, Dict, Any, Tuple, Iterator, Iterable
+
+
+def login_with_user_token(username: str,
```

Comment:
method name `login_with_user_token` is incorrect - `login with credentials` or `get user token` are more correct